src/AbstractHyperrectangle.jl

```diff
@@ -159,3 +160,35 @@ function ∈(x::AbstractVector{N},
     end
     return true
 end
+
+"""
+    ⊆(H1::AbstractHyperrectangle, H2::AbstractHyperrectangle)::Bool
+
+Check whether a given hyperrectangle is contained in another hyperrectangle.
+
+### Input
+
+- `H1` -- first hyperrectangle (containee?)
```

Comment:
"inner hyperrectangle" vs. "outer hyperrectangle"   ??

Reply:
Well, this is what the function has to find out. I did not find a good way to express this.